std-bits/base/src/main/java/org/enso/base/statistics/LinearModel.java

```diff
@@ -0,0 +1,4 @@
+package org.enso.base.statistics;
+
+public record LinearModel(double slope, double intercept, double rSquared) {
+}
```

Comment:
```suggestion
public record LinearModel(double slope, double intercept, double rSquared) {}
```

This is subjective but isn't it nicer to have the `{}` on one line if the record has no 'contents'?